Makefile

```diff
@@ -33,6 +34,11 @@ build-bridge:
 build-kernel:
 	@cargo build --package jstz_kernel --target wasm32-unknown-unknown $(PROFILE_OPT)
 
+.PHONY: build-jstzd-kernel
+build-jstzd-kernel: build-kernel
+	@cp target/wasm32-unknown-unknown/$(PROFILE_TARGET_DIR)/jstz_kernel.wasm $(JSTZD_KERNEL_PATH)
+
+# TODO: Remove once jstzd replaces the sandbox
```

Comment:
Could you track the `TODO` with a linear ticket please?

Reply:
done